### ISSUE TITLE
Proposed Business 57 — Classic Literature Translation Studio Phase 1 visual UI reference

### ISSUE THREAD

**Issue by skerishKang** [skerishKang/ai-revenue-lab#217]
## Purpose

Create the Phase 1 visual UI reference for proposed Business 57 — **Classic Literature Translation Studio / 고전문학 번역실**.

Parent queue: #154
Product decision: #216

This is a polished static visual reference using public-domain or repository-local synthetic material. It is not accepted UX, a production translator, a model-training pipeline, a publishing service or a backend.

## Product promise

> 저작권이 만료되었거나 정식 허락된 고전소설과 고전시를, 원전의 의미·어조·모호성·운율을 최대한 보존한 번역과 현대 한국어 독해를 위한 번역으로 나란히 편집하고, 중요한 번역 판단과 근거를 독자가 직접 검토하게 한다.

## Primary visual direction

Use **Parallel Literary Edition / 나란히 읽는 번역판**.

The UI should feel like a serious literary edition, translator’s desk and annotated reading room—not an AI chat, generic document translator, SaaS dashboard or settings panel.

Visual characteristics:

- Korean-first editorial reading surface;
- source text, fidelity translation and modern-reading translation aligned without becoming a spreadsheet;
- warm off-white paper, deep ink, restrained oxblood or indigo annotation accent;
- fine rules, folio numbers, edition marks, marginalia, glossary tabs and proofreader symbols;
- serif-led literary typography with Korean system-serif fallbacks and restrained system sans for controls;
- generous line height and measure suitable for long-form reading;
- materially different visual treatment for source, translated prose, uncertainty and editorial notes;
- no gradients, glassmorphism, rounded-card overload, neon AI motifs, chat bubbles or generic robot imagery.

No external font CDN or font binaries.

## Required reference research

Before implementation, inspect and document current official or primary references for:

- public-domain source discovery and bibliographic provenance;
- Korean Copyright Commission guidance on copyright duration and derivative translations;
- WIPO/Berne principles for translation and adaptation rights;
- literary parallel-text editions and critical editions;
- translator workspaces or publishing proof systems;
- poetry translation layouts preserving stanza, line break, sound and annotations;
- U.S. Copyright Office or equivalent official material on generative-AI training and copyrighted corpora as a comparative risk reference.

Also review 3–5 adjacent products or editorial references, then record:

- adopted patterns;
- rejected patterns;
- why this product differs from Google Translate, DeepL, generic AI chat, Language Learning Magazine, Living Fiction and AI Dubbing Studio.

Record the research in:

```text
reference/business-57-classic-literature-translation-studio-v1/REFERENCE_NOTES.md
```

## Source and rights fixture

Use one verified public-domain classical novel excerpt as the primary fixture and one verified public-domain poem as the secondary fixture.

Preferred approach:

- original-language source from a recognized public archive or critical source;
- author death/publication details and territory/date status documented;
- Korean translations authored specifically for this reference;
- no copied modern Korean translation;
- no existing translator’s wording used as the default answer;
- no long third-party commentary copied into the interface.

The implementation report must explain why each selected source is usable and distinguish the public-domain original from any separately protected modern translation.

### Primary — 원전 보존 번역

The default screen must emphasize:

- source meaning and ambiguity;
- narrator distance and tense;
- repeated images and terms;
- deliberately preserved difficult syntax where meaningful;
- translator notes for consequential choices.

### Secondary — 현대 독해 번역

This may modernize syntax and vocabulary, but the UI must reveal:

- what changed;
- why it changed;
- what nuance may have been weakened or strengthened;
- whether the change affects tone, rhythm, ambiguity or cultural distance.

### Translation lenses

Do not present named living writers, translators or screenwriters as imitation modes.

Use non-attributive controls such as:

- 장중한 문어체;
- 담담하고 건조한 리듬;
- 짧고 절제된 문장;
- 장면 중심의 긴장;
- 서정적 이미지 밀도;
- 고어 보존;
- 현대적 대화;
- 주석 밀도.

A visible disclosure must state that these are editorial characteristics, not replicas or endorsements of named authors.

### 1. Translation library / 번역 서가

Show a small curated library of public-domain or licensed works with:

- cover or typographic title treatment;
- author and original language;
- source/rights status;
- current edition state;
- default mode `원전 보존 번역`;
- novel and poetry distinction.

Avoid streaming-service cards or ecommerce shelves.

### 2. Source and fidelity spread / 원문·원전 보존본

This is the hero state.

Show one novel passage with:

- original-language source;
- aligned Korean fidelity translation;
- selected sentence linkage;
- marginal key-term notes;
- visible uncertainty or alternative rendering;
- restrained reading controls.

The user should immediately understand that this is a literary edition, not a raw machine-translation result.

### 3. Two-edition comparison / 두 번역판 비교

Compare the same passage as:

- 원전 보존 번역;
- 현대 독해 번역.

Show material changes with editorial annotations, not red/green code diffs. At least four differences should cover syntax, word choice, ambiguity and dialogue register.

### 4. Translation decision ledger / 번역 판단 기록

Show a reviewable list or folio of consequential decisions:

- source phrase;
- selected rendering;
- alternatives considered;
- semantic and stylistic rationale;
- confidence or unresolved state;
- human-review status.

This must feel editorial and evidence-based rather than like model telemetry.

### 5. Poetry edition / 고전시 번역판

Show one poem with:

- original stanza and line structure;
- literal rendering;
- poetic Korean rendering;
- rhythm, sound, image and untranslatable-element notes;
- visible line-break decisions.

Do not compress poetry into prose cards.

### 6. Mobile reading edition / 모바일 읽기

At actual `390×844`, show a credible long-form reading view with:

- one active translation edition;
- a compact source reveal;
- readable line length;
- footnote or margin-note access without horizontal overflow;
- clear source/rights and AI-assistance disclosure.

### 7. Signature motion — Translation Weave / 번역 결 엮기

Create one restrained `680ms` signature motion showing how selected source fragments connect to the chosen Korean rendering and then settle into the final paragraph.

Requirements:

- source and final translation geometry remain stable;
- no whole-page fade or layout jump;
- only linking rules, emphasis marks or annotation layers animate;
- focus and scroll position remain stable;
- replay is deterministic;
- `prefers-reduced-motion: reduce` shows the complete final state immediately.

## Content requirements

The reference should include:

- at least one meaningful ambiguity with two plausible Korean renderings;
- one culturally specific term that requires explanation;
- one repeated source word tracked across multiple sentences;
- one deliberately difficult sentence retained in the fidelity edition;
- one modernized dialogue or syntax choice in the modern-reading edition;
- one explicit statement of what was lost or changed;
- one poetry example where sound/rhythm and literal meaning conflict;
- model/corpus/source/rights disclosure;
- human editorial review state.

All Korean translations must be newly authored for this reference and must not be represented as professional published translations.

## Image and asset direction

The UI is text-led but should include restrained original/local visual assets such as:

- edition cover or title page;
- source manuscript/page abstraction;
- translator pencil marks or proof symbols;
- archival library seal;
- poetry image motif derived from the source without copying protected artwork.

Use original repository-local SVG/WebP assets or properly licensed public-domain assets. No hotlinks or runtime CDN.

Create:

```text
reference/business-57-classic-literature-translation-studio-v1/IMAGE_SOURCES.md
```

For every asset record local path, source URL or original basis, creator/rightsholder, licence or public-domain basis, verification date, intended state and `reference-only` status.

## Accessibility and responsive requirements

Validate at:

- desktop `1440×1100`;
- tablet `768×1024`;
- mobile `390×844`;
- reduced-motion desktop.

Required:

- horizontal overflow `0`;
- keyboard state navigation;
- visible focus;
- correct language attributes for source passages;
- accessible names for translation-mode and note controls;
- sufficient contrast for source, translation, uncertainty and notes;
- no information conveyed only by colour;
- no console errors, page errors, failed local assets or external runtime requests.

## Required evidence

Commit reproducible text evidence under:

```text
reference/business-57-classic-literature-translation-studio-v1/evidence/**
```

Include:

- static contract validation;
- browser validation script;
- validation JSON;
- exact commands;
- deterministic CSS/JS version check;
- source and rights verification notes;
- screenshots for the key desktop states and actual mobile state;
- Translation Weave motion evidence;
- reduced-motion final-state evidence.

Binary evidence may be archived separately, but filenames, byte sizes and SHA-256 values must be recorded.

## Allowed scope

```text
reference/business-57-classic-literature-translation-studio-v1/**
```

Do not modify:

- `apps/**`;
- another Business workspace;
- Business Registry or candidate backlog;
- repository-wide operations documents;
- CI or deployment configuration.

## Target branch

```text
feat/business-57-classic-literature-translation-studio-ui
```

## Target Draft PR title

```text
feat(classic-literature-translation): add Phase 1 visual UI reference
```

## Phase boundary

This issue is `UI_ONLY`.

Do not implement:

- actual model training or fine-tuning;
- retrieval over unlicensed copyrighted books;
- production translation generation;
- named living-writer imitation;
- uploads, accounts, persistence or collaboration;
- publishing, printing, billing or subscriptions;
- UX acceptance or backend APIs;
- Cloudflare deployment before `UI_APPROVED` and separate user authorization;
- Ready transition, merge or Issue closure without Web CTO review.

## Proposed deployment after approval

```text
Cloudflare Pages project:
ai-revenue-business-57-classic-literature-translation-studio

Production URL:
https://ai-revenue-business-57-classic-literature-translation-studio.pages.dev/
```

This is only a proposed target. Do not create or deploy it until the exact UI head is `UI_APPROVED` and the user separately authorizes deployment.

## Acceptance criteria

- all seven visual states are present and materially distinct;
- the source/fidelity spread is the dominant product state;
- modern-reading translation remains visibly secondary and comparable;
- novel and poetry treatment are both credible;
- named-author imitation is absent from controls and copy;
- source, translation and rights provenance are explicit;
- required viewports and reduced motion pass;
- all source changes remain inside the allowed workspace;
- Draft PR remains open and unmerged;
- final status is `UI_REVIEW_READY`, not `UI_APPROVED`.

**Comment by skerishKang** [comment]
## Product-relationship update — B57 remains standalone; connect to B61 via interoperability

Superseding relationship decision: **do not absorb B57 into B61 and do not retire B57 solely because StoryMemory can display translated classics.**

New cross-product authority: #1146

B57 remains the specialist Translation Studio and keeps ownership of:
- source/fidelity/modern-reading comparison;
- ambiguity/alternatives;
- glossary/repeated-term tracking;
- translation decision ledger;
- loss/simplification disclosure;
- human editorial review;
- source/rights/provenance.

B61 remains StoryMemory and may consume an accepted, versioned B57 translation edition for Reader / Memory / AI presentation.

Intended bidirectional UX:

```text
B57 -> Open in StoryMemory
B61 -> Open in Translation Studio
```

The Phase-1 B57 UI/provenance in this Issue therefore remains useful as B57's own product evidence. It is **not** a migration source to be wholesale merged into StoryMemory. PR #220/#430 remain governed by their existing B57 lifecycle until separately decided.

No current B61 MVP dependency is introduced by this update.

**Comment by skerishKang** [comment]
Implemented — capability verified in main. Triage verdict: IMPLEMENTED (evidence anchor in #2018 batch table; sample set CTO-verified: PR #1585 merged 2026-09-02, 7 module files present, 44 reference folders present). Closed per CTO execution of the #2018 triage.
